crates/subspace-node/src/chain_spec.rs

```diff
@@ -221,6 +223,8 @@ pub fn mainnet_compiled() -> Result<GenericChainSpec, String> {
                     ])
                     .expect("Number of elements is below configured MaxRewardPoints; qed"),
                 },
+                domain_block_pruning_depth: 14_400u32,
+                staking_withdrawal_period: 14_400u32,
```

Comment:
And here.

Reply:
I explained about why I do not prefer constants here.

Its more explicit to see what is the number and even if we introduce constant, it will have to be in the same file

Reply:
If we misspell a named constant, the compiler will complain. We can also put that constant in a primitives crate, so there's only one default value. (Which is then used by the production genesis config.)

If we mistype a number, then the compiler will accept the value. This is a production bug risk. So do we need to add tests that make sure these duplicate numbers are always the same?

The numbers will be different in test code and dev/test networks, but that's less risky than production code. So I don't really mind what we do outside production.

Reply:
Right now there is only one source of truth i.e this value.
Not point making this a constant since if someone changes the value here or the value of the constant it does not matter.